bool enable_nwctrl 
---
bool status  
